src/server.h

```diff
@@ -2121,6 +2122,9 @@ struct valkeyServer {
     int reply_buffer_resizing_enabled; /* Is reply buffer resizing enabled (1 by default) */
     /* Local environment */
     char *locale_collate;
+    /* io_uring */
+    int io_uring_enabled; /* If io_uring enabled (0 by default) */
+    struct io_uring *io_uring;
```

Comment:
There is no need to expose the io_uring pointer in the server struct. Let's remove it

Reply:
@PingXie It needs to be stored somewhere. Is a separate global variable better?

Reply:
Singleton instance returned by io_uring, client need to get the io_uring instance and passed through pointer?